charts/openstack-cloud-controller-manager/values.yaml

```diff
@@ -130,3 +130,7 @@ extraVolumeMounts:
 # cluster name that used for created cluster
 cluster:
   name: kubernetes
+
+clusterRoleName: system:cloud-controller-manager
+
+serviceAccountName: cloud-controller-manager
```

Comment:
hm. I'm a bit skeptical about removing the `openstack-` suffix, since it may cause inconsistencies during an upgrade. But if this was already merged into master, I'm fine with this. @zetaab @dulek what do you think?

Reply:
Yes, that is why it is important to include the release notes. 

Once users install this the RBAC should be like it should have been in the first place.

Reply:
Just to give a bit more context from #2332: The client is initialised as `cloud-controller-manager` which is why a service account of the same name is created in the `kube-system` system.

The helm chart currently creates RBAC for an `openstack-cloud-controller-manager` one, which results in broken permissions for every installation that runs with `--use-service-account-credentials=true`.

You can find additional details in the original PR.